docs/content/_index.md

```diff
@@ -15,7 +15,7 @@ notoc: true
 - [What providers does rclone support?](#providers)
 - [Download](/downloads/)
 - [Install](/install/)
-{{< rem MAINPAGELINK >}}
+  {{< rem MAINPAGELINK >}}
```

Comment:
This change is probably wrong, see:

https://github.com/rclone/rclone/blob/c37fe733dfd0c69ccad35fa9d4441f99133e2006/bin/make_manual.py#L152

Reply:
PS: I don't see the donate link appearing on https://rclone.org, so that re.sub does not seem to be working as intended @ncw..

Reply:
I think it is working - the extra link is in [MANUAL.md](https://raw.githubusercontent.com/rclone/rclone/refs/heads/master/MANUAL.md) - the frontpage doesn't need it as it is in the sidebar.